idseq_dag/util/custom_marshal.py

```diff
@@ -0,0 +1,8 @@
+import shelve
```

Comment:
Looks like custom_marshal and the other marshal are accidental duplicates.

Reply:
thx